ee/desktop/runtime/runner.go

```diff
@@ -119,16 +119,10 @@ func (r *DesktopUsersProcessesRunner) Interrupt(interruptError error) {
 		r.procsWg.Wait()
 	}()
 
-	signal := os.Interrupt
-	// os.Interrupt is not supported on windows, so use os.Kill instead
-	if runtime.GOOS == "windows" {
-		signal = os.Kill
-	}
-
 	for uid, proc := range r.uidProcs {
-		if err := proc.process.Signal(signal); err != nil {
+		if err := sendShutdownCommand(proc.process.Pid); err != nil {
```

Comment:
Might want to signal _after_ sendShutdownCommand

Reply:
It will a kill signal after the sendShutdownCommand if the proc is still running after the time out has elapsed. Is this what you meant?

Reply:
Yeah, something like that. If the process doesn't exit cleanly when we send it a shutdown command, what's next?

Reply:
we are doing that, a little lower in the `for select` statement we send a kill after a tmeout